python/example_code/bedrock-runtime/models/anthropic/claude3_chatbot_demo/0_get_models.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
Add licensing boiler plate throughout (to each file)

Reply:
done